src/panel/category/CategoryPanel.jsx

```diff
@@ -103,8 +105,23 @@ export default class CategoryPanel extends React.Component {
 
   fetchData = async () => {
     const { category, query } = this.props.poiFilters;
-
     const bbox = window.map.mb.getBounds();
+
+    // On desktop, fetch PoIs on a bbox that doesn't include the list panel's width and the header's height
+    if (!isMobileDevice()) {
```

Comment:
Shouldn't we do the same on mobile by taking the top bar and panel into account?